Laptop Price Predictor/Model/Laptop_Price_Prediction.py

```diff
@@ -0,0 +1,57 @@
+# LAPTOP PRICE PREDICTION
+
+# Importing Libraries
+
+import pandas as pd
+from sklearn import preprocessing
+from statsmodels.stats.outliers_influence import variance_inflation_factor
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.metrics import r2_score
+
+# Getting our Data
+
+df = pd.read_csv(r'./Laptop Price Predictor/Dataset/laptoppricing.csv', encoding = 'unicode_escape')
+
+# Data Preprocessing
+
+df.rename(columns={'ï»¿Manufacturer':'Manufacturer'}, inplace=True)
```

Comment:
on writing this simple code, I am not getting any special characters for manufacturer:

```python
import pandas as pd

df = pd.read_csv(r"Laptop Price Predictor\Dataset\laptoppricing.csv")
print(df.columns)
```
Output:
```bash
Index(['Manufacturer', 'IntelCore(i-)', 'IntelCoreGen',
       'processing speed(GHz)', 'Ram(gb)', 'HDD(gb)', 'SSD(gb)',
       'Graphics(gb)', 'ScreenSize(inch)', 'Price'],
      dtype='object')
```
Just read the dataset without any encoding